src/arcaflow_plugin_sdk/test_schema.py

```diff
@@ -963,6 +963,8 @@ class TestData1:
                 default="", metadata={"id": "test-field", "name": "G"}
             )
             I: typing.Any = None
+            J: typing.List[typing.Dict[str, str]] \
+                = dataclasses.field(default_factory=list)
 
         schema.test_object_serialization(
             TestData1(A="Hello world!", B=5, C={}, D=[], H=3.14),
```

Comment:
Does/should `J` be _referenced_ anywhere (like at line 970)?

Reply:
Nevermind:  [Dave already asked this](https://github.com/arcalot/arcaflow-plugin-sdk-python/pull/141#discussion_r1834643510).